client/src/game/ui/CardLayout.ts

```diff
@@ -135,7 +135,7 @@ export default class CardLayout extends Konva.Group {
             opacity: 1,
             easing: Konva.Easings.EaseOut,
             onFinish: () => {
-              if (!card || !layoutChild) {
+              if (card === undefined || layoutChild === undefined) {
```

Comment:
add both null checks

Reply:
those can't be null, sorry :)

Reply:
their types are HanabiCard and Konva.Node, none of which is nullable.